Tests/Functional/FunctionalTests.xml

```diff
@@ -29,6 +29,14 @@
             <directory suffix="Test.php">./</directory>
         </testsuite>
     </testsuites>
+    <coverage>
+        <include>
+            <!--
+                This path needs an adaption in extensions, when coverage statistics are wanted.
+            -->
+            <directory>../../../../../../typo3/sysext/*/Classes/</directory>
```

Comment:
I see wee have the same path for unit tests.
Maybe I don't understand how this works, but I would expect we need to define the Classes folder of EXT:tea instead of foreign code.
Can you explain why we include the system extensions instead of our own Classes folder?

Reply:
I've done some more research: It turns out we don't need to provide any paths here - having an empty `coverage` node is enough to make PHPUnit happy with the configuration file.